.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "eslint.enable": false
```

Comment:
@Jvaleti11 generally any application specific code is not to be checked in. The way around this is that when you're committing and you see this. Add that file type to .gitignore. Eg add ".vscode/" to .gitignore

I'll get you to do this on this PR.